Tests/scripts/hook_validations/script.py

```diff
@@ -82,90 +71,83 @@ def _get_arg_to_required_dict(cls, script_json):
                         (arg in old_args_to_required and required != old_args_to_required[arg]):
-                    print_error("You've added required args in the script file '{}', the field is '{}'".format(
-                        self.file_path, arg))
+                    print_error(Errors.added_required_fields(self.file_path, arg))
                     return True
-
         return False
 
     def is_there_duplicates_args(self):
+        # type: () -> bool
+
         """Check if there are duplicated arguments."""
-        args = [arg['name'] for arg in self.current_script.get('args', [])]
+        args = [arg['name'] for arg in self.current_file.get('args', [])]
         if len(args) != len(set(args)):
             return True
-
         return False
 
     def is_arg_changed(self):
+        # type: () -> bool
         """Check if the argument has been changed."""
-        current_args = [arg['name'] for arg in self.current_script.get('args', [])]
-        old_args = [arg['name'] for arg in self.old_script.get('args', [])]
+        current_args = [arg['name'] for arg in self.current_file.get('args', [])]
+        old_args = [arg['name'] for arg in self.old_file.get('args', [])]
 
         if not self._is_sub_set(current_args, old_args):
-            print_error("Possible backwards compatibility break, You've changed the name of an arg in "
-                        "the file {}, please undo.".format(self.file_path))
+            print_error(Errors.breaking_backwards_arg_changed(self.file_path))
             return True
-
         return False
 
     def is_context_path_changed(self):
+        # type: () -> bool
         """Check if the context path as been changed."""
-        current_context = [output['contextPath'] for output in self.current_script.get('outputs', [])]
-        old_context = [output['contextPath'] for output in self.old_script.get('outputs', [])]
+        current_context = [output['contextPath'] for output in self.current_file.get('outputs', [])]
+        old_context = [output['contextPath'] for output in self.old_file.get('outputs', [])]
 
         if not self._is_sub_set(current_context, old_context):
-            print_error("Possible backwards compatibility break, You've changed the context in the file {0},"
-                        " please undo.".format(self.file_path))
+            print_error(Errors.breaking_backwards_context(self.file_path))
             return True
-
         return False
 
     def is_docker_image_changed(self):
+        # type: () -> bool
         """Check if the docker image as been changed."""
         # Unnecessary to check docker image only on 5.0 and up
-        if server_version_compare(self.old_script.get('fromversion', '0'), '5.0.0') < 0:
-            old_docker = get_dockerimage45(self.old_script)
-            new_docker = get_dockerimage45(self.current_script)
+        if server_version_compare(self.old_file.get('fromversion', '0'), '5.0.0') < 0:
```

Comment:
lets use `LooseVersion`

Reply:
changed server_version_compare to use LooseVersion.
You're making my life much worse than they are.
Peace and love